### PR TITLE
Create go template

### PR DESCRIPTION
**Pull request by firstmorecoffee**
Generate a go template with same features as the python and typescript ones.

### CODE REVIEW

File: sym-aws-go-approvals/approve/main.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-lambda-go/lambda"
+
+	"github.com/serverless-templates/sym-aws-go-approvals/internal"
```
Comment (yasyf) [inline sym-aws-go-approvals/approve/main.go:9]: shouldn't this be `/symopsio/serverless-templates/internal`?

File: sym-aws-go-approvals/expire/main.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-lambda-go/lambda"
+
+	"github.com/serverless-templates/sym-aws-go-approvals/internal"
```
Comment (yasyf) [inline sym-aws-go-approvals/expire/main.go:9]: ditto

File: sym-aws-go-approvals/internal/sym.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import (
+	"fmt"
+
+	"github.com/symopsio/types/go/enums"
+	"github.com/symopsio/types/go/models"
+)
+
+// GetIdentity is a utility function to get a user identity for a given service
+func GetIdentity(user *models.User, service enums.Service) (*models.User_Identity, error) {
```
Comment (yasyf) [inline sym-aws-go-approvals/internal/sym.go]: maybe `FindIdentity`? I expected `GetIdentity` to do a fetch of some kind

File: sym-aws-go-approvals/serverless.yml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  name: aws
+  runtime: go1.x
+
+# you can overwrite defaults here
+  endpointType: REGIONAL
+
+# makes serverless variables double braced so as to not conflict with CF
+  variableSyntax: '\${{(((?!\${{).)+?)}}' 
+
+# you can add statements to the Lambda function's IAM Role here
```
Comment (yasyf) [inline sym-aws-go-approvals/serverless.yml:16]: is this comment relevant still, or leftover from the template?

File: sym-aws-go-approvals/approve/main.go (hunk 1 above)
Comment (firstmorecoffee) [reply sym-aws-go-approvals/approve/main.go:9]: nah this is the module path as generated in `gomod.sh`

File: sym-aws-go-approvals/serverless.yml (hunk 4 above)
Comment (firstmorecoffee) [reply sym-aws-go-approvals/serverless.yml:16]: Still useful IMO for people starting with this template

File: sym-aws-go-approvals/internal/sym.go (hunk 3 above)
Comment (firstmorecoffee) [reply sym-aws-go-approvals/internal/sym.go]: changed

**Review by yasyf — APPROVED** [review]

**Pull request merged**